### DISCUSSION TITLE
sb500 random streaming freezes on mac vlc player

### DISCUSSION THREAD

**Discussion by eberhardhummel** [GerryDazoo/Slinger#520] [General]
experiencing random video freeze on mac vlc player. Sometimes it freezes after 10 seconds, sometimes it can go an hour or more without freezing. Streaming client and slingbox server are separate machines. All components are on gb ethernet wired. Tried with two different sb500s and same result.

steps to reproduce:
(1) sb500 component in rca audio
(2) streaming client mbp monterey 12.01 vlc 3.0.17.3 (Intel 64bit) quad core i7 - 16gb ram 500gb ssd
(3) slingbox server mbp monterey 12.6.1 -  quad core i7 - 16gb ram 500gb ssd, Python 3.9.6
(4) config.ini attached

startup log:
python3 slingbox_server.py
Version : 3.08f Running on macOS-12.6.1-x86_64-i386-64bit pid= 937 slingbox_server.py
Using config file config.ini
Connection Manager Running on port 49001 with 10 max streams using URL slingbox.
Warning TCP buffering might not be sufficent for reliable streaming
Warning TCP buffering might not be sufficent for reliable streaming
Streamer Running:  10 config.ini SLINGBOX 49001 49001 7340032
starting up on  port 49001 7340032
Reading Custom Remote definition from remote_directv-h24.txt
 * Serving Flask app 'slingbox_server'
 * Debug mode: off
49001 No valid slingbox ip info found in config.ini
Finding Slingbox on local network. My IP Info =  192.168.253.222
Slingbox Found 192.168.253.210 5201 " poolhouse500 " FinderID ****
Finding Slingbox on local network. My IP Info =  192.168.1.106
49001 Using slingbox at  ('192.168.253.210', 5201)
Streamer:  49001 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
01/24/2023, 09:05:47.940   Streaming request from ('192.168.253.160', 49001)

Slinginfo  "350/500/M1/M2" 12 30 192.168.253.210 5201 3072 10
49001 Starting Stream for  192.168.253.160:49001
Connecting... ('192.168.253.210', 5201) Control
49001 Using encrypted password: ****
49001 Selecting Video Source 1
Keycodes= [1, 2, 3, 4, 5, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 21, 24, 27, 28, 29, 30, 31, 33, 35, 36, 38, 39, 40, 41, 42, 46, 62, 63, 64]
VideoParameters: Resolution= 12 FrameRate= 30 VideoBandwidth= 2000 VideoSmoothness= 63 IframeRate= 5 AudioBitRate= 64
Connecting... ('192.168.253.210', 5201) Stream
49001 Stream started at 01/24/2023, 09:05:55.916  744 0
........01/24/2023, 09:07:26.955  49001 1 Clients.192.168.253.160:49001 
........01/24/2023, 09:08:58.232  49001 1 Clients.192.168.253.160:49001 
........01/24/2023, 09:10:30.180  49001 1 Clients.192.168.253.160:49001 
........01/24/2023, 09:12:00.207  49001 1 Clients.192.168.253.160:49001 
......01/24/2023, 09:13:18.081  49001 Stream Terminated for  192.168.253.160:49001
.49001 Shutting down connections
49001 Logging Out
Streamer:  49001 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox

[config.ini.txt](https://github.com/GerryDazoo/Slinger/files/10576869/config.ini.txt)

**Comment by GerryDazoo** [comment]
VLC for MAC is problematic. Try a different player.
